src/style.css

```
@@ -0,0 +1,140 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: normal;
+  color: #060b35;
+  font-size: 10px;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  width: 200px;
+  height: 408px;
+}
+
+.card__mac-image {
+  width: 160px;
+  height: 134px;
+
+  margin-top: 32px;
```

You're main element card should place youre inner elements instead add padding to .card
```suggestion
.card {
padding:32px 16px 16px;
}
```